canonical/store-components
# added children to be displayed above the main footer

## Done
Added children (4 columns) to be displayed above the main footer

## QA steps
Go to https://store-components-23.demos.haus/?path=/story/footer--default-story
Check if there are links in the 4 columns displayed above the main footer

## Fixes
Fixes: https://warthogs.atlassian.net/browse/WD-1617
